internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -69,15 +71,33 @@ func (e *EnvStackConfig) Template() (string, error) {
 		return "", err
 	}
 
+	if e.AdjustVpcConfig.VpcCIDR == nil {
+		e.AdjustVpcConfig.VpcCIDR = aws.String(defaultVPCCIDR)
+	}
+	if e.AdjustVpcConfig.PublicSubnetCIDRs == nil {
+		e.AdjustVpcConfig.PublicSubnetCIDRs = aws.StringSlice(strings.Split(defaultPublicSubnetCIDRS, ","))
+	}
+	if e.AdjustVpcConfig.PrivateSubnetCIDRs == nil {
+		e.AdjustVpcConfig.PrivateSubnetCIDRs = aws.StringSlice(strings.Split(defaultPrivateSubnetCIDRS, ","))
+	}
+
 	content, err := e.parser.Parse(EnvTemplatePath, struct {
```

Comment:
Is there any reason we might want this to be its own defined struct? for reusability or ease of modifying the code?

Reply:
I think it is for reusability and ease of extending the struct (by categorizing fields), since other than vpc resources we'll have more options available and if flattening those two it would be very long.